dmsite/backend/dmsite/db_manager/db_manager.py

```diff
@@ -9,7 +9,6 @@
 dynamodb = boto3.resource('dynamodb', region_name='us-east-1', endpoint_url="http://dynamodb.us-east-1.amazonaws.com")
 ERR_STR = "There was an error trying to access the AWS database. See 'errorStr' for details."
 
-
```

Comment:
I'm noticing a lot of extra lines that're being removed. A lot of them were added to get PyCharm to adhere to PEP8 style guidelines. No issues removing them, figured it'd be working mentioning.

Reply:
Oh, we should definitely be following PEP8, I must've deleted them accidentally when fixing merge issues